libs/eo/metering-points/shell/src/lib/eo-metering-points-shell.component.ts

```diff
@@ -20,27 +20,59 @@ import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
 
 import { WATT_CARD } from '@energinet-datahub/watt/card';
 import { WattToastService } from '@energinet-datahub/watt/toast';
+import { WattValidationMessageComponent } from '@energinet-datahub/watt/validation-message';
 
 import { EoMeteringPointsTableComponent } from './eo-metering-point-table.component';
 import { EoMeteringPointsStore } from './eo-metering-points.store';
-import { EoBetaMessageComponent } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
-  imports: [AsyncPipe, NgIf, EoMeteringPointsTableComponent, WATT_CARD, EoBetaMessageComponent],
+  imports: [
+    AsyncPipe,
+    NgIf,
+    EoMeteringPointsTableComponent,
+    WATT_CARD,
+    WattValidationMessageComponent,
+  ],
   selector: 'eo-metering-points-shell',
   styles: [
     `
-      watt-card-title {
+      :host {
         display: flex;
-        gap: var(--watt-space-m);
-        align-items: center;
+        flex-direction: column;
```

Comment:
Vi flexer nu? hvorfor? var der problemer med det andet før?

Reply:
Stylingen er blot ændret, vi laver ikke længere noget i forhold til `watt-card` men selve containeren. Her benytter vi flex for at lave mellemrum mellem info-beskeden i toppen og tabellen på siden.